opfpush/src/main/java/org/onepf/opfpush/OPFPushHelper.java

```diff
@@ -211,6 +209,11 @@ private void sendMessage(@NonNull final Message message) {
         }*/
     }
 
+    /**
+     * Returns the registration ID of current provider or {@code null} if there isn't current provider.
+     *
+     * @return registration ID of current provider.
```

Comment:
Better way to document this would probably be something like this:

@return registration ID if there's currently registered push provider, null otherwise.